<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devTab</class>
 <widget class="QWidget" name="devTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_0x01">
      <attribute name="title">
       <string>线圈</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_0x02">
      <attribute name="title">
       <string>离散量</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_0x03">
      <attribute name="title">
       <string>保持寄存器</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_0x04">
      <attribute name="title">
       <string>输入寄存器</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
